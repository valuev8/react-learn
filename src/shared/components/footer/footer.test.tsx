import { render } from '@testing-library/react';
import React from 'react';
import Footer from './footer';

describe('Footer', () => {
  it('Should render', () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  })
})
