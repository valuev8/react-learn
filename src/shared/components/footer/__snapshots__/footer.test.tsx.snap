// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render 1`] = `
<DocumentFragment>
  <div
    class="sc-AxirZ daYQmo"
  >
    <a
      class="sc-AxjAm PldMZ"
      href="#"
    >
       Cyber Movies 
    </a>
  </div>
</DocumentFragment>
`;
